benchmark/add.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Make sure to create a .npmrc file
+# registry=http://localhost:4873
+
+PACQUET="../target/release/pacquet add fastify"
+PNPM="pnpm add fastify --silent"
+YARN="yarn add fastify --silent"
+BUN="bun add fastify --no-cache --no-save --silent"
+
+FILE_CLEAN="rm -rf package.json pnpm-lock.yaml node_modules .yarn yarn.lock .pnp* && echo {} > package.json"
+PNPM_CLEAN="(pnpm cache prune || true)"
```

Comment:
there is no such command in pnpm. There is `pnpm store prune`

Reply:
👍